### ISSUE TITLE
ConnectionError returned if my Internet network is not stable

### ISSUE THREAD

**Issue by renjunxiang** [huggingface/pytorch-pretrained-BERT#222]
Hi,
although I have download BERT pretrained model,  "ConnectionError" returned if my Internet network is not very stable.
Function ```file_utils.cached_path``` needs stable internet. 
Is there any way to avoid checking for amazonaws before loading bert-embedding?

**Comment by cmeister747** [comment]
I'm guessing you're using some of the classes defined in modeling.py, such as one of the Bert "pretrained models" (e.g. any of the models that inherit from `PreTrainedBertModel`)? On construction, each of these classes takes a `config` argument, where `config` is a BertConfig object (also defined in modeling.py). The BertConfig can either be created from a model at one of the links in `PRETRAINED_MODEL_ARCHIVE_MAP` or from a config file stored in a local directory. You just have to set the `pretrained_model_name` to a local directory containing a bert_config.json file and a pytorch_model.bin file rather than one of 'bert-base-uncased', 'bert-large-uncased' etc. Setting `pretrained_model_name` to one of the latter options will try to pull from the Amazon AWS repositories. So if you're running the run_classification.py script, you would set the 'bert-model' flag to the directory with your downloaded bert model if you don't want it to pull from AWS. One thing is if you've downloaded one of the original Google Bert models,  you'll need to convert  tf checkpoints to pytorch bin files. There's a script for this in the repository. You shouldn't need to worry about this if you've downloaded one of the models at the links in `PRETRAINED_MODEL_ARCHIVE_MAP` (defined at the top of modeling.py)

TLDR: Set `--bert-model` to the directory with your downloaded Bert model

Does that make any sense?

**Comment by renjunxiang** [comment]
Yes! Set local directory in modeling.py and tokenization.py can solve my problem. Thank you so much!
